apps/finance/app/src/lib/token-utils.js

```diff
@@ -38,3 +43,24 @@ export const tokenDataFallback = (tokenAddress, fieldName, networkType) => {
   }
   return fallbacksForNetwork.get(addressWithoutChecksum)[fieldName] || null
 }
+
+export async function getTokenSymbol(app, address) {
+  let tokenAbi = [].concat(tokenBalanceOfAbi, tokenDecimalsAbi, tokenSymbolAbi)
+  let token = app.external(address, tokenAbi)
+  let tokenSymbol = await token
```

Comment:
We should probably add comments here that we currently don't need a `try/catch` block here because aragon.js doesn't throw the error back at us.

That being said, I was thinking we could program a bit defensively and use a `try/catch` anyway so it won't break when aragon.js actually does return errors:

```js
let tokenSymbol

try {
  // try string
  tokenSymbol = await token.symbol()
  ...
  // Note: aragon.js currently doesn't throw errors when a call fails (returns undefined instead) so we manually throw
  if (!tokenSymbol) {
     throw new Error()
  }
} catch () {
  // try bytes32
  try {
    tokenSymbol = await token.symbol()
    ...
    tokenSymbol ? toUtf8(tokenSymbol) : null
  } catch () {
     // ignore
  }
}

return tokenSymbol
```